storage/buckets/main.go

```diff
@@ -15,7 +15,8 @@
 // Sample buckets creates a bucket, lists buckets and deletes a bucket
 // using the Google Storage API. More documentation is available at
 // https://cloud.google.com/storage/docs/json_api/v1/.
-package main
+
+package buckets
```

Comment:
```suggestion
// +build ignore

package buckets
```

This will make it so this file doesn't build and hopefully fix the compilation issue.

Reply:
Thanks, it helps